### PR TITLE
Must-C speech translation corpus

### PR DESCRIPTION
**Pull request by hirofumi0810**

### CODE REVIEW

File: egs/must_c/st1/conf/tuning/decode_pytorch_transformer.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+batchsize: 0
+beam-size: 10
+penalty: 0.0
+maxlenratio: 0.0
+minlenratio: 0.0
+ctc-weight: 0.5
```
Comment (ShigekiKarita) [inline egs/must_c/st1/conf/tuning/decode_pytorch_transformer.yaml]: I do not think this config is needed

**Comment by sw005320** [comment]
This looks very good for me.
Since this PR only changes the recipe part, we can merge this one to the master branch instead of v.0.6.0. What do you think, @hirofumi0810?

**Comment by hirofumi0810** [comment]
> * we may think of generalizing `divide_lang.sh` and moving to `https://github.com/espnet/espnet/tree/master/utils`
> * `remove_duplicate_utterances.py` seems to be a very good utility script. Again we may think of moving it to `https://github.com/espnet/espnet/tree/master/utils`

Is it OK to change other recipes in this PR to achieve this?

**Comment by hirofumi0810** [comment]
> This looks very good for me.
> Since this PR only changes the recipe part, we can merge this one to the master branch instead of v.0.6.0. What do you think, @hirofumi0810?

Please merge into master.

**Comment by sw005320** [comment]
> > * we may think of generalizing `divide_lang.sh` and moving to `https://github.com/espnet/espnet/tree/master/utils`
> > * `remove_duplicate_utterances.py` seems to be a very good utility script. Again we may think of moving it to `https://github.com/espnet/espnet/tree/master/utils`
> 
> Is it OK to change other recipes in this PR to achieve this?

Maybe we can do it in the other PRs. Let's focus on the current PR :)

**Comment by hirofumi0810** [comment]
> * how `egs/must_c/st1/local/copy_data_dir.sh` is differnt from the original `copy_data_dir.sh`? Could you add the comment?

This script accepts text with case information like data/text.**.
But this is not used in st1 and thus I removed it from this PR.

**Comment by hirofumi0810** [comment]
This is ready to be merged.

**Comment by sw005320** [comment]
Thanks, @hirofumi0810!

**Pull request merged**